Spanner/src/Instance.php

```diff
@@ -482,12 +482,19 @@ public function createDatabaseFromBackup($name, $backup, array $options = [])
      * $database = $instance->database('my-database');
      * ```
      *
+     * ```
+     * Database role configured on the database object
+     * will be applied to the session created by this object.
+     * $database = $instance->database('my-database', ['databaseRole' => 'Reader']);
```

Comment:
```suggestion
     * Database role configured on the database object
     * will be applied to the session created by this object.
     * ```
     * $database = $instance->database('my-database', ['databaseRole' => 'Reader']);
```

Reply:
Done